ml-agents/tests/yamato/yamato_utils.py

```diff
@@ -150,10 +150,11 @@ def checkout_csharp_version(csharp_version):
         return
 
     csharp_tag = f"com.unity.ml-agents_{csharp_version}"
-    csharp_dirs = ["com.unity.ml-agents", "Project"]
+    csharp_dirs = ["com.unity.ml-agents", "com.unity.ml-agents.extensions", "Project"]
```

Comment:
If we don't sync the extensions package, we'll get compile failures because the current extensions code is not compatible with old package code.